addons/account/models/account_move.py

```diff
@@ -3525,17 +3525,18 @@ def _get_last_sequence_domain(self, relaxed=False):
     def _get_starting_sequence(self):
         # EXTENDS account sequence.mixin
         self.ensure_one()
-        year_part = "%04d" % self.date.year
+        account_date = self.date or self.invoice_date or fields.Date.context_today(self)
```

Comment:
```suggestion
        date = self.date or self.invoice_date or fields.Date.context_today(self)
```
Not sure `account_date` is appropriate in this context, since it could lead to think it's related to the move's account.
To avoid any confusion, I think `date` would be fine (or `move_date`?).

Reply:
> Not sure `account_date` is appropriate in this context, since it could lead to think it's related to the move's account. To avoid any confusion, I think `date` would be fine (or `move_date`?).

The code has been changed according to your suggestion. Thank you :)